digits/dataset/images/classification/test_views.py

```diff
@@ -66,7 +66,7 @@ class BaseViewsTestWithImageset(BaseViewsTest):
     IMAGE_WIDTH     = 10
     IMAGE_CHANNELS  = 3
     BACKEND         = 'lmdb'
-    COMPRESSION     = 'none'
+    COMPRESSION     = 'none' 
```

Comment:
My text editor (visual slick) has an option to automatically remove trailing spaces.